src/ocm/hooks/useClusterSupportedPlatforms.ts

```diff
@@ -41,7 +47,8 @@ export default function useClusterSupportedPlatforms(clusterId: string) {
   }, [error, addAlert, clusterId, alerts]);
 
   return {
-    isPlatformIntegrationSupported,
+    isPlatformIntegrationSupported: supportedPlatformIntegration !== undefined,
+    supportedPlatformIntegration: supportedPlatformIntegration as SupportedPlatformIntegrationType,
```

Comment:
This should work the other way around. If `isPlatformIntegrationSupported` is true then `supportedPlatformIntegration` should hold a useful value.

Also the logic assigning the value to `supportedPlatformIntegration` is not correct now. The API could return something like: `['nutanix', 'vsphere']`. In this case the platform integration is not supported! Although, through the API might be possible to discover different hosts from different VM platforms, in the UI we allow integration only when all the hosts are originated from the same platform vendor (either Nutanix or VMware/vSphere).

Reply:
Based on what we discussed, I assumed the platform integration must be unique. Meaning there cannot be mixed hosts of different platforms. So how would the UI know which of the platform is needs to use for the integration, if there is more than 1? The `cluster.platform.type` can hold a single value